test/util/testnode/full_node_test.go

```diff
@@ -27,10 +34,29 @@ func (s *IntegrationTestSuite) SetupSuite() {
 	if testing.Short() {
 		s.T().Skip("skipping full node integration test in short mode.")
 	}
+	t := s.T()
+
+	cparams := DefaultParams()
+	cparams.Block.MaxBytes = appconsts.MaxShareCount * appconsts.ContinuationSparseShareContentSize
 
-	s.T().Log("setting up integration test suite")
+	accounts := make([]string, 40)
+	for i := 0; i < 40; i++ {
+		accounts[i] = tmrand.Str(10)
+	}
 
-	s.accounts, s.cctx = DefaultNetwork(s.T())
+	ecfg := encoding.MakeConfig(app.ModuleEncodingRegisters...)
+	blobGenState := blobtypes.DefaultGenesis()
+	blobGenState.Params.GovMaxSquareSize = appconsts.MaxSquareSize
+	cctx, _, _ := NewNetwork(
+		t,
+		cparams,
+		DefaultTendermintConfig(),
+		DefaultAppConfig(),
+		accounts,
+		SetBlobParams(ecfg.Codec, blobGenState.Params),
+	)
+	s.cctx = cctx
+	s.accounts = accounts
 }
 
 func (s *IntegrationTestSuite) Test_Liveness() {
```

Comment:
I don't know how this tests liveness. Wouldn't we just want one block to be produced. I also don't understand why this is flaky with fast block times

Reply:
we can probably get rid of it and just wait one block. iirc this was just a copy of https://github.com/celestiaorg/cosmos-sdk/blob/57a798a56219a6581a78f94976f592b88ea3aa01/testutil/network/network_test.go#L37-L40